starter-code/src/TodaysFoods.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import foods from './foods.json'
+
+
+class TodaysFoods extends Component {
+
+    state = {
+        todaysFoods: [
+            {
+                name: "Pizza",
+                quantity: 0,
+                calories: 0,
+            },
+        ]
+    }
+
+
+    render() {
+
+
+
+        return (<div>
+
+            {this.state.todaysFoods.map((food, idx) => {
+                return <div>
+                    <p>{food.name}</p>
+                    <p> {food.calories}</p>
+                    <p> {food.quantity}</p>
+
+                </div>
```

Comment:
Maybe create a component here as well to easily make it prettier afterwards. This seems fine for now but when you add classes and some more markup I would favor an additional component.